src/components/KTable/KTable.vue

```diff
@@ -554,9 +554,32 @@ export default defineComponent({
           ...rowListeners,
           ...cellListeners,
           click(e: any) {
-            const isPopoverContent = e.target.className.includes('k-popover')
+            const targetClasses = e.target.className
+            let isIgnored = ignoredElements.includes(e.target.tagName.toLowerCase())
+            let isPopoverContent = false
+
+            // check for popover class
+            if (typeof targetClasses === 'string' || Array.isArray(targetClasses)) {
+              isPopoverContent = targetClasses.includes('k-popover')
+            } else if (typeof targetClasses === 'object') {
+              isPopoverContent = Object.keys(targetClasses).includes('k-popover')
+            }
+
+            // check parent for popover class
+            if (e.target.closest('.k-popover-content') !== null) {
+              isPopoverContent = true
+            }
```

Comment:
Just checking, should this always eval to true if the previous if block is still false?

Reply:
Yes